initfiles/etc/DIR_NAME/environment.xml.in

```diff
@@ -15,10 +15,9 @@
   <runtime>${RUNTIME_PATH}</runtime>
   <sourcedir>${CONFIG_SOURCE_PATH}</sourcedir>
   <user>${RUNTIME_USER}</user>
-  <ports>
```

Comment:
Configmgr does not create a ports section when creating new configs, but will put what is in the conf file into the Environment section (without nesting).  To make it so a hand edit to the xml is not required for this setting with configs generated by configmgr it was decided to flatten the ports subsection.  @garonsky can elaborate.  Another route is to add to configmgr to support nested sections from the conf file but that seems a more involved method.